go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -199,6 +199,40 @@ func (oc *BaseSecondaryNetworkController) Reconcile(netInfo util.NetInfo) error
 	return util.ReconcileNetInfo(oc.ReconcilableNetInfo, netInfo)
 }
 
+func (oc *BaseSecondaryNetworkController) FilterResource(objType reflect.Type, obj interface{}) bool {
+	if !oc.IsPrimaryNetwork() {
+		return true
+	}
```

Comment:
why this check?

Reply:
I removed it but I had to modify the multinet policy unit tests, @pperiyasamy @npinaeva ptal

Reply:
yes @kyrtapz , that was a bug with unit test, both namespaces must be having same NAD config referring to same network to get multinet policy to be really effective for the network.